### ISSUE TITLE
Best real time cloning? 

### ISSUE THREAD

**Issue by WYNNGATE** [CorentinJ/Real-Time-Voice-Cloning#1165]
Hi, I'm looking for the best voice changer, in other words, my speech to a cloned voice.

Use case is for youtube and other VO

**Comment by gabrielmontagne** [comment]
This commercial solution  is quite good, I've tried it a lot and it's great.   It's still not fully open, but you can request a test drive, https://www.resemble.ai/speech-to-speech/

**Comment by Lolagatorade** [comment]
Really sucks there is AI for everything you can run in local hardware. AI art stable diffusion, text GPT you got alpaca and llama. But nothing good for voice cloning.

**Comment by thedarthvader2000** [comment]
ElevenLabs in terms of quality has a really effective voice cloning in my opinion. Does anyone have a guess / know what their training protocol may have been? So the base model, and then what else they added to it to bring it to where it is today. Breaking it down is the first step to making an open source alternative which I’m very interested in doing!

**Comment by WYNNGATE** [comment]
Not real time but surprisingly great,

https://play.google.com/store/apps/details?id=com.voicecopy.app

**Comment by Lolagatorade** [comment]
> ElevenLabs in terms of quality has a really effective voice cloning in my opinion. Does anyone have a guess / know what their training protocol may have been? So the base model, and then what else they added to it to bring it to where it is today. Breaking it down is the first step to making an open source alternative which I’m very interested in doing!

Honestly, there's not much things that are open in terms of Voice cloning. You can go on GitHub and type in voice, cloning and search for whatever comes up I believe some of those results have research papers. I remember there was some Chinese repository that has it running locally but of course I don't know Chinese. I just find it very strange how open image generation face swap, and all the other things are, but there's so many companies that are private when it comes to Voice cloning

**Comment by WYNNGATE** [comment]
Indeed! Yes, very strange. I suspect that ai singing will take over soon.
Sigh...it will be great to hear such perfect singing voices..but at what
price? The further destruction of culture etc.. I will say, I was very
impressed with the app
https://play.google.com/store/apps/details?id=com.voicecopy.app


PLEASE let me know if you discover any quality real time solutions.
Voicemod seems maybe okay, looks like it can be ran on a windows server and
then controlled via Android app.. still, nothing too great.
